### ISSUE TITLE
message.reply() takes no parameters - how to specify reply body?

### ISSUE THREAD

**Issue by VBobCat** [vgrem/Office365-REST-Python-Client#484]
This library is fantastic. But I have a question:

I want to execute the operation [message: reply as defined in MS-Graph API](https://docs.microsoft.com/en-us/graph/api/message-reply?view=graph-rest-1.0&tabs=http). It expects a JSON body specifying reply properties and a comment:

```
POST https://graph.microsoft.com/v1.0/me/messages/AAMkADA1MTAAAAqldOAAA=/reply
Content-Type: application/json

{
  "message":{  
    "toRecipients":[
      {
        "emailAddress": {
          "address":"samanthab@contoso.onmicrosoft.com",
          "name":"Samantha Booth"
        }
      },
      {
        "emailAddress":{
          "address":"randiw@contoso.onmicrosoft.com",
          "name":"Randi Welch"
        }
      }
     ]
  },
  "comment": "Samantha, Randi, would you name the group please?" 
}
```

However, the Message.reply() method (message.py, line 105) takes no parameters, so how can I specify this body?

Thank you very much.

**Comment by vgrem** [comment]
Greetings,

thank you for catching this inconcitency with API,

**Comment by vgrem** [comment]
Greetings,

thank you for catching this inconsistency with API, `reply` method support for `comment` and `message` parameters has been introduced  (will be included in `2.3.12` release shortly)

Here is an example: 


```python
client = GraphClient(acquire_token_by_username_password)

comment = "Fanny, would you join us next time?"
message = client.me.messages.new()  # type: Message
message.to_recipients = ["fannyd@contoso.onmicrosoft.com"]
primary_message.reply(message, comment).execute_query()
```

**Comment by VBobCat** [comment]
Thank you, sir. I'm glad if I helped.

Em sáb., 5 de mar. de 2022 às 15:23, Vadim Gremyachev <
***@***.***> escreveu:

**Comment by vgrem** [comment]
The change has been implemented and available in version 2.3.12
